packages/app/src/sandbox/worker/utils.ts

```diff
@@ -0,0 +1,163 @@
+/* eslint-disable no-console */
+
+import { invariant } from 'outvariant';
+// @ts-ignore
+import SandpackWorker from 'worker-loader?publicPath=/&name=sw.[hash:8].worker.js!./sw';
+import { CHANNEL_NAME, IWorkerPingMessage } from './types';
+
+/**
+ * TODO: Find a way to get the SW file name from webpack
+ */
+const __SERVICE_WORKER_BUNDLE_NAME = SandpackWorker.toString()
+  .replace('function() {\n  return new Worker("/" + "', '')
```

Comment:
@DeMoorJasper, this is a hack to get the service-worker file URL. How can I properly get the URL from webpack without any hack? I found a way in line 5 to import the file, but I need this same path in a variable to use I feel like.